examples/crate_universe/cargo_remote/BUILD.names.bazel

```diff
@@ -27,6 +27,7 @@ rust_binary(
     version = "0.12.0",
     deps = all_crate_deps(normal = True) + [
         ":names",
+        "@crate_index_cargo_remote//:clap",  # Clap is optional dependency and has to be added manually
```

Comment:
I'm not sure at the moment, what this example is testing.

If I understand correctly, the `clap` crate is an optional dependency of the `names` crate. And in this case it is not supposed to be enabled by default.

Therefore we need to specify it manually.

Reply:
The issue here is that `clap` is an optional dependency. But it is enabled by `default` feature.

And prior to my changes it was resolved and added to dependencies set.

After my changes, the dependency is resolved, but it is filtered out on the later stage, when we remove all optional dependencies.

I assume that the "defalut" features should be working out of the box. I'm looking into how to fix it.